java-library/src/main/java/com/microsoft/azure/functions/sql/annotation/SQLInput.java

```diff
@@ -17,6 +17,11 @@
 @Target(ElementType.PARAMETER)
 @CustomBinding(direction = "in", name = "sql", type = "sql")
```

Comment:
Wait - isn't the name here what you need to update? Or does having a name property in the interface override the one set here?

Reply:
Yes, having the name property without a default will require users to always specify a name.